lib/templ.ml

```diff
@@ -353,6 +429,9 @@ let templ_eval_var conf =
   | ["just_friend_wizard"] -> VVbool conf.just_friend_wizard
   | ["friend"] -> VVbool conf.friend
   | ["manitou"] -> VVbool conf.manitou
+  | ["plugin"; plugin] ->
+      let list = List.map (fun p -> Filename.basename p) conf.plugins in
+      VVbool (List.mem plugin list)
```

Comment:
you can remove the List.map step with: `List.exists (fun p -> Filename.basename p = plugin) conf.plugins`